src/dotnet/APIView/APIViewWeb/Helpers/CodeFileHelpers.cs

```diff
@@ -557,6 +557,13 @@ private static void BuildTokensForDiffNodes(CodePanelData codePanelData, CodePan
                         {
                             var parentNode = codePanelData.NodeMetaDataObj[parentNodeIdHashed];
                             parentNode.IsNodeWithDiffInDescendants = true;
+
+                            if ((diffTokenRowResult.Before.Any() && !beforeRowClasses.Contains("doc")) || 
+                                (diffTokenRowResult.After.Any() && !afterRowClasses.Contains("doc")))
+                            {
+                                parentNode.IsNodeWithNoneDocDiffInDescendants = true;
```

Comment:
Is "none doc diff" meant to mean, "has no doc diff" ? If so, it's a bit easier to reason about if you invert it as `IsNodeWithDocDiff` similar to the `IsNodeWithDiff` naming.

Reply:
That would work except `IsNodeWithDocDiff ` is not really interesting because it can also have none doc diff.